android/src/androidTest/java/com/getcapacitor/android/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.getcapacitor.android;
+
+import static org.junit.Assert.*;
+
+import android.content.Context;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.platform.app.InstrumentationRegistry;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+
+    @Test
+    public void useAppContext() throws Exception {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
+
+        assertEquals("com.getcapacitor.android", appContext.getPackageName());
+    }
+}
```

Comment:
Why are we adding this template test file? Ideally, we should be adding proper test cases. But, if we aren't, I don't see the point of adding this.

Reply:
This applies to all template test classes within the PR.